src/test/java/com/feedzai/commons/sql/abstraction/engine/impl/abs/EngineGeneralTest.java

```diff
@@ -2619,6 +2619,45 @@ public void updateWithWhereTest() throws DatabaseEngineException {
         );
     }
 
+    @Test
+    public void updateFrom1ColTest() throws DatabaseEngineException {
+        test5Columns();
+        test1Column();
```

Comment:
Since it is just for this one test, I'd create the "TEST2" table right here instead of a new method...